src/Core/MailTemplates/Mjml/build.js

```diff
@@ -41,8 +41,10 @@ if (!fs.existsSync(config.outputDir)) {
   }
 }
 
-// Find all MJML files with absolute path
-const mjmlFiles = glob.sync(`${config.inputDir}/**/*.mjml`);
+// Find all MJML files with absolute paths, excluding components directories
+const mjmlFiles = glob.sync(`${config.inputDir}/**/*.mjml`, {
```

Comment:
Error: 

<img width="2210" height="814" alt="Image" src="https://github.com/user-attachments/assets/49326480-6e12-4f03-aed1-d59c4331ba98" />

This validation error occurs because the build script treats this shared subcomponent as a standalone component. We have this subcomponent here because only the AC team needs it. By moving the files into an AC directory, we can reduce code review friction.

This change will make the build script recognize all “components” directories as non-standalone MJML templates.

Result:

<img width="1448" height="704" alt="Image" src="https://github.com/user-attachments/assets/b87497e3-a866-411e-8d15-ea8609cd64fc" />